docs/topics/db/optimization.txt

```diff
@@ -213,8 +213,10 @@ Understand :meth:`~django.db.models.query.QuerySet.select_related` and
 them:
 
 * in :doc:`managers and default managers </topics/db/managers>` where
-  appropriate. Be aware when your manager is and is not used; sometimes this is
-  tricky so don't make assumptions.
+  appropriate. Be aware when your manager is and is not used; for example,
+  related object access uses the base manager rather than the default manager,
+  which may bypass custom manager methods. See
+  :doc:`/topics/db/managers` for details.
```

Comment:
This is identical to the link just before this. Instead, I think we can link to the `using-managers-for-related-object-access` anchor on that page.